cmd/celvet/main.go

```diff
@@ -31,7 +33,8 @@ import (
 
 func main() {
 
-	humanReadable := flag.BoolP("human-readable", "r", true, "print out values in human-readable formats")
+	humanReadable := flag.BoolP("human-readable", "r", true, "print out values in human-readable formats (only applies if --json/-j is not passed)")
```

Comment:
Why disallow -r for json? It's not immediately obvious to me why that wouldn't be allowed.

Reply:
At present `-r` only affects `CostError.HumanReadableError()`, which is only used during text output, so it technically could be used in conjunction with JSON, but it wouldn't do anything.

Reply:
`-r` still documentation refers to `--json` which doesn't exist any more.

Reply:
Whoops, fixed the documentation referring to the old `--json` flag and clarified that it applies only to text output.